ChatQnA/docker/gaudi/README.md

```diff
@@ -263,3 +281,23 @@ To access the frontend, open the following URL in your browser: http://{host_ip}
 ```
 
 ![project-screenshot](../../assets/img/chat_ui_init.png)
+
+Here is an example of running ChatQnA:
+
+![project-screenshot](../../assets/img/chat_ui_response.png)
+
+## 🚀 Launch the Conversational UI (react)
```

Comment:
Displaying two UIs here might confuse users. It's necessary to include a description stating that users only need to choose one of them.

Reply:
Done